### DISCUSSION TITLE
Program crashed！！！！！！

### DISCUSSION THREAD

**Discussion by 815943555** [filecoin-project/lotus#3923] [General Q&A] [answered]
Please see attachment
[out.log](https://github.com/filecoin-project/lotus/files/5246129/out.log)

**Comment by jennijuju** [comment]
Whats your lotus version and hardware spec?

also see if [this](https://filecoinproject.slack.com/archives/C017CCH1MHB/p1597338404043600?thread_ts=1597338129.041000&cid=C017CCH1MHB) helps you

**Comment by skyqhxiao** [comment, marked as answer]
Try this, 

If you are building Lotus 0.7.1 and have an older Intel or AMD processor, set:

export CGO_CFLAGS="-D__BLST_PORTABLE__"
